recipes/dataconf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dataconf" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dataconf-{{ version }}.tar.gz
+  sha256: c4f845b405a4b5ddbf59267879dd5a9e1a1b4ee73fff9c0e0df5ca560f8b2705
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - dataconf = dataconf.cli:run
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - pyhocon >=0.3.54,<0.4.0
+    - python >=3.6,<4.0
+    - python-dateutil >=2.8.1,<3.0.0
+
+test:
+  imports:
+    - dataconf
+  commands:
+    - pip check
+    - dataconf --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zifeo/dataconf
+  summary: Simple dataclasses configuration management for Python with hocon/json/yaml/properties/env-vars/dict support.
+  license: MPL-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - MattChanTK
```

Comment:
Why is this not `thewchan`?

Reply:
good catch, not sure why that's the default from grayskull... sometimes I forget to change it